sentry-spring-boot-starter/src/test/kotlin/io/sentry/spring/boot/jdbc/SentryDsProxyAutoConfigurationTest.kt

```diff
@@ -0,0 +1,41 @@
+package io.sentry.spring.boot.jdbc
+
+import io.sentry.dsproxy.SentryQueryExecutionListener
+import io.sentry.spring.boot.SentryAutoConfiguration
+import javax.sql.DataSource
+import kotlin.test.Test
+import org.assertj.core.api.Assertions
+import org.springframework.boot.autoconfigure.AutoConfigurations
+import org.springframework.boot.test.context.FilteredClassLoader
+import org.springframework.boot.test.context.runner.ApplicationContextRunner
+
+class SentryDsProxyAutoConfigurationTest {
```

Comment:
a Fixture here would be useful, big part of the tests are duplicated

Reply:
Test got refactored but without introducing Fixture as I think its too simple for that.